agents/specialized/filesystem_agent.py

```
@@ -5,91 +5,97 @@
 
             return {
-                'success': True,
-                'path': os.path.relpath(path, self.base_path),
-                'items': items,
-                'timestamp': datetime.utcnow().isoformat()
+                "success": True,
+                "path": os.path.relpath(path, self.base_path),
+                "items": items,
+                "timestamp": datetime.utcnow().isoformat(),
             }
         except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)